android/src/main/java/com/reactnativestripesdk/CollectBankAccountLauncherFragment.kt

```diff
@@ -32,6 +34,18 @@ class CollectBankAccountLauncherFragment(
 ) : Fragment() {
   private lateinit var collectBankAccountLauncher: CollectBankAccountLauncher
 
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+
+    val stripeSdkModule: StripeSdkModule? = context.getNativeModule(StripeSdkModule::class.java)
+    if (stripeSdkModule != null && stripeSdkModule.eventListenerCount > 0) {
```

Comment:
let's do something like this instead: https://github.com/stripe/stripe-react-native/blob/447804f5529fe91c2f6ffbf983d3151ece283ba4/android/src/main/java/com/reactnativestripesdk/PaymentSheetFragment.kt#L143-L148

Reply:
The event listener is optional, so it wouldn’t be right to throw an error in this case.

My thinking here was: If we have an event listener in React, then this _might_ be a Financial Connections event listener, so let’s try to send the even across the bridge.